cmd/kubeadm/app/cmd/config.go

```diff
@@ -211,7 +211,7 @@ func getDefaultedInitConfig() (*kubeadmapi.InitConfiguration, error) {
 		LocalAPIEndpoint: kubeadmapiv1beta2.APIEndpoint{AdvertiseAddress: "1.2.3.4"},
 		BootstrapTokens:  []kubeadmapiv1beta2.BootstrapToken{placeholderToken},
 		NodeRegistration: kubeadmapiv1beta2.NodeRegistrationOptions{
-			CRISocket: constants.DefaultDockerCRISocket, // avoid CRI detection
+			CRISocket: constants.UnknownCRISocket, // avoid CRI detection
```

Comment:
so i think we should keep this DefaultDockerCRISocket because it's still the default in the API.
in this case it just skips the detection.

getDefaultedInitConfig() is used to fetch API defaults for `config print*`
[1]